src/ResourceManager/Storage/Commands.Management.Storage/Commands.Management.Storage.csproj

```diff
@@ -63,8 +63,8 @@
     <Reference Include="Microsoft.WindowsAzure.Storage, Version=9.0.0.0, Culture=neutral, PublicKeyToken=31bf3856ad364e35, processorArchitecture=MSIL">
       <HintPath>..\..\..\packages\WindowsAzure.Storage.9.0.0\lib\net45\Microsoft.WindowsAzure.Storage.dll</HintPath>
     </Reference>
-    <Reference Include="Newtonsoft.Json, Version=10.0.0.0, Culture=neutral, PublicKeyToken=30ad4fe6b2a6aeed, processorArchitecture=MSIL">
```

Comment:
@maddieclayton, I have changed the PR to not reference json.
But I still need some time to test if this will fix the issue.

I am currently blocked by a build failure on my machine so can't test it.
Will update the PR when get the test result.

Reply:
Okay, I will mark as do not merge for now - please let me know when you are ready for me to take a look again.